style-sheets/primevue-components-overrides.scss

```diff
@@ -391,7 +392,57 @@ label {
 .#{variables.$primevue-prefix}-card .#{variables.$primevue-prefix}-card-footer {
     padding: 0;
 }
+
 /* ------------------------------------ primevue table --------------------------------- */
+
+.custom-access-table {
+    background: transparent;
+    border-radius: 0.25rem 0.25rem 0 0;
+    border: 0.125rem solid map.get(lightTheme.$light-theme, 'border-subtle-00');
+}
+
+.custom-table-header {
+    padding: 1rem 1rem 1.5rem;
+    h3 {
+        @extend %heading-03;
+        margin: 0;
+        padding: 0;
+    }
+
+    span {
+        @extend %body-compact-01;
+        margin: 0;
+        padding: 0;
+        color: map.get(lightTheme.$light-theme, 'text-secondary');
+    }
+}
+
+.span-icon {
+    margin-left: 0.9375rem;
+}
+
+.remove-action {
+    color: map.get(lightTheme.$light-theme, 'text-error');
+}
+
+.#{variables.$primevue-prefix}-datatable
+.#{variables.$primevue-prefix}-sortable-column
+.#{variables.$primevue-prefix}-sortable-column-icon {
+    display: none;
+}
+
+.#{variables.$primevue-prefix}-input-icon-right {
```

Comment:
If this class is for input, then let's place it under the input component section, not under the datable section

Reply:
Will do, thanks!

Reply:
Hi Jota, I saw you copied this to the input section, shall we remove it from here then?

Reply:
Hello! Agreed, I will do it ASAP!